game/src/challenges/cutscene.rs

```diff
@@ -168,20 +168,18 @@ fn make_panel(
     make_task: &Box<dyn Fn(&mut EventCtx) -> Widget>,
     idx: usize,
 ) -> Panel {
-    let mut prev = app
+    let prev = app
         .cs
-        .btn_plain_dark_icon("system/assets/tools/prev.svg")
+        .btn_plain_dark_icon("system/assets/tools/circled_prev.svg")
         .image_dims(45.0)
         .hotkey(Key::LeftArrow)
-        .bg_color(Color::CLEAR, ControlState::Disabled);
-    if idx == 0 {
-        prev = prev.disabled();
-    }
-    let prev = prev.build_widget(ctx, "back");
+        .bg_color(Color::CLEAR, ControlState::Disabled)
+        .disabled(idx == 0)
+        .build_widget(ctx, "back");
 
     let next = app
         .cs
-        .btn_plain_dark_icon("system/assets/tools/next.svg")
+        .btn_plain_dark_icon("system/assets/tools/circled_next.svg")
```

Comment:
I added actual svg assets for the next/prev icons. 
<img width="389" alt="Screen Shot 2021-01-21 at 12 00 46 PM" src="https://user-images.githubusercontent.com/217057/105392009-4eeeb200-5be0-11eb-9725-c9f1e8eeade3.png">


This one we were using in cutscene, is a little different, and a bit much for most pagination, so I gave the asset a more specific name:

<img width="350" alt="Screen Shot 2021-01-21 at 12 00 09 PM" src="https://user-images.githubusercontent.com/217057/105391910-37afc480-5be0-11eb-8f84-6ba22a819f62.png">

As a side note, a "pager" widget or builder function might be nice some day. We use it in a half dozen places, and some of them seem a little "off" in undesirable ways.

Reply:
> I added actual svg assets for the next/prev icons.

Good call, this is robust to future changes we make about fonts.

> a "pager" widget or builder function might be nice some day

I'll keep it in mind. Cutscenes, the tutorial instructions, tables... probably a few other places.